gempy/core/data.py

```diff
@@ -1234,9 +1241,19 @@ def set_default_c_o(self, range_var=None):
 
         """
         if range_var is None:
-            range_var = self.df['range']
+            range_var = self.df.loc['values', 'range']
+
+        # if type(range_var) is pn.Series:
```

Comment:
code blocks should be removed, git history keeps it

Reply:
Yes, I always leave it as reference while I am coding the new solution and half of the time I leave it there. I have to be more strict with it. Thanks for caching it